### DISCUSSION TITLE
How to move with Ctrl+p and Ctrl+n in FindUsage and FindInpath Windows

### DISCUSSION THREAD

**Discussion by alohaeee** [JetBrains/ideavim#903] [Q&A]
I can't get best workflow with my ideavim config. When i opening findusage window ideavim keybindings turns off

This map doesn't help neither
```
map <c-p> :action Tree-selectPrevious<cr>
map <c-n> :action Tree-selectPrevious<cr>
```

**Comment by citizenmatt** [comment]
IdeaVim is an editor extension, so maps only work inside the editor. Unfortunately, there is no way to make this work inside the Find Usages window.

**Comment by alohaeee** [comment]
Okay, got it. IdeaVim has NerdTree plugin that work inside project pane so i guess this is possible to make working ideavim bindings inside other windows. Right now fixed this issue with changing bindings in IDE directly

**Reply by citizenmatt** [reply]
Yeah, NerdTree is special, with explicit support for the Project view. It's not really possible to make that general enough to work in any possible tool window, so there are currently no plans to support Vim keystrokes in anything other than editors.
